esm/legacy-client.browser.js

```diff
@@ -5,7 +5,7 @@ import { initLegacyClient } from './api/v2/legacy-client.js';
 import { prefixUrl } from './prefix-url.js';
 import { request } from './request.fetch.js';
 
-export function initCleverClient (config) {
+export function initCleverClient (config, onError) {
```

Comment:
**question:** nothing wrong here but could you provide some context about why we may need this now? (an example of what it allows)

The code itself is ok for me, but I'm missing the point.

Reply:
It allows to pass a function to handle errors when there's an API call from the legacy client. We need to do it this way as we don't have another way to handle it after the init.

Reply:
Maybe, a little bit of JsDoc may help to understand what to pass here: an optional function